nvm.sh

```diff
@@ -1907,6 +1908,64 @@ nvm_download_artifact() {
   nvm_echo "${TARBALL}"
 }
 
+nvm_tar_knows_J() {
+  ERR_MSG=$(tar -J 2>&1 | head -1)
```

Comment:
maybe this would be better, since we know for sure that openbsd won't support it?
```suggestion
  if [ "$(nvm_get_os)" = 'openbsd' ]; then
    return 1;
  fi
  ERR_MSG=$(tar -J 2>&1 | head -1)
```

Reply:
Hm, not sure man. We just moved away from OS verification.

Reply:
lol yes, i'm mainly thinking about not *limiting* the scope of the fix to bsd. but i guess it's valid that if bsd starts shipping `-J` support, we'd want it to start working

Reply:
@ljharb sorry for late reply. In that case, we still don't need the suggested change, correct?

Reply:
yeah, agreed, let's keep it this way for now